docs/reference/protocol-spec.md

```diff
@@ -85,13 +86,23 @@ Redis generally uses RESP as a [request-response](#request-response-model) proto
 * The server replies with a RESP type.
   The reply's type is determined by the command's implementation and possibly by the client's protocol version.
 
-We categorize every RESP data type as either _simple_ or _aggregate_.
-Simple types are similar to scalars in programming languages that represent plain literal values, for example, Booleans.
-Aggregates, such as Arrays and Maps, can have varying numbers of sub-elements and nesting levels.
+RESP is **always** ASCII-encoded (UTF-8/Latin-1).
```

Comment:
I picked it up from #2203 :) Too much?

Reply:
Well, in that case I think it can be explained whenever the spec mentions a character, it is a byte in the ASCII interpretation (e.g. \r and \n are the byte values 13 and 10), but anything else is binary data. (UTF-8 and Latin-1 only matters for byte values > 127, but those are never used in the protocol, apart from as binary data.)

Reply:
Idea: In the beginning of the document, for example here, describe when RESP is concerned with characters and when it isn't.


```suggestion
Whenever this specification mentions the word "character", it is to be understood as the ASCII interpretation of a single byte. In other situations, such as in the contents of bulk strings, RESP is not concerned with characters and character encodings. The payload of bulk strings can contain any binary data, any sequence of bytes, including null bytes.
```

Reply:
I propose: "RESP is a **BINARY** protocol using control sequences encoded using standard **ASCII** (Example: the character A is encoded with the binary byte of value 65, the characters <CR>, <LF>, <SP> with 13, 10 and 32 respectively.)"